runhouse/servers/http/http_server.py

```diff
@@ -1153,13 +1170,13 @@ async def main():
         cluster_config["server_connection_type"] = "tls" if use_https else "none"
 
     await obj_store.aset_cluster_config(cluster_config)
-    logger.info("Updated cluster config with parsed argument values.")
 
     await HTTPServer.ainitialize(
         default_env_name=default_env_name,
         conda_env=conda_name,
         enable_local_span_collection=use_local_telemetry
         or configs.data_collection_enabled(),
+        logs_level=parse_args.log_level,
```

Comment:
i think we need to be case sensitive here, would just confirm that the previous change to the CLI arg enforces this is uppercase